project/src/components/offers-list/offers-list.tsx

```diff
@@ -1,18 +1,22 @@
 import CityCard from '../../components/city-card/city-card';
-import {offerType} from '../../types/types';
 import {useState} from 'react';
+import {offerType} from '../../types/types';
+import {useAppSelector} from '../../hooks';
 
-type OffersListProps = {
-  offers: offerType[];
-}
-
-function OffersList({offers}: OffersListProps):JSX.Element {
+function OffersList():JSX.Element {
   const [cityHoveredId, setCityHoveredId] = useState(undefined as string | undefined);
+  const {city, offers} = useAppSelector((state) => state);
+  const placesToStay:offerType[] = [];
+
+  offers.forEach((offer) => offer.city.name === city ? placesToStay.push(offer) : undefined);
+
+  // eslint-disable-next-line no-console
+  console.log(placesToStay);
 
   return (
     <section className='cities__places places'>
       <h2 className='visually-hidden'>Places</h2>
-      <b className='places__found'>312 places to stay in Amsterdam</b>
+      <b className='places__found'>{offers.length} places to stay in {offers[0].city.name}</b>
```

Comment:
```suggestion
      <b className='places__found'>{offers.length} places to stay in {offers[0].city.name}</b>
```
а если будет `offers.length === 0`, то все сломается!!! А!!!